CMakeLists.txt

```diff
@@ -155,8 +155,8 @@ message("Building with Qt${QT_DEFAULT_MAJOR_VERSION} (${Qt5Core_VERSION_STRING})
 
 # OpenCV
 find_package(
-        OpenCV 4.5
-        COMPONENTS core calib3d aruco video videoio highgui imgproc
+        OpenCV 4.10
```

Comment:
Wasn't there an `EXACT` for the version here at some point in time?

Reply:
Yes, good catch! I removed it during local testing and apparently forgot to add it back 🙈 (tested with 4.8 from vcpkg registry before I tested with 4.10)

Though one could discuss whether it is needed. While PeTrack will not reproduce test results with a newer version, most of the time it will work just fine using a newer version.

But yeah, feel free to add it back 😅

Reply:
For me, it's fine. I just noticed this when I was looking at the changes.

Reply:
🤔 Then I think we can leave it like that...